src/components/VulnerabilityEdgeAlert.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Alert, Button } from '@patternfly/react-core';
+import { InProgressIcon } from '@patternfly/react-icons';
+const VulnerabilityEdgeAlert = ({
+  activeAlert,
+  navigateProp,
```

Comment:
This here should not be needed to be passed down, the components, InventoryDetails & VulnerabilityAlert, are in the same router context and calling it here directly will yield the same `navigate` behaviour. 

Use `useNavigate` here directly.